### PR TITLE
pods - topic specific dedicated servers

### PR DESCRIPTION
**Pull request by ishandutta2007**
The pods server is down for many months due to heavy traffic. This may finally get it up and running again.
@timgrossmann : I have added 
contact.timgrossmann@gmail.com and help.instapy@gmail.com as owners for each of the servers.

This change also has a minor renaming of topic `beauty` to `fashion `

### CODE REVIEW

File: instapy/pods_util.py [hunk 1]
```diff
@@ -3,10 +3,24 @@
 from .settings import Settings
 from .database_engine import get_database
 
+def get_server_endpoint(topic):
+    if topic=='fashion':
```
Comment (breuerfelix) [inline instapy/pods_util.py]: whitespace is not your enemy !

Comment (ishandutta2007) [reply instapy/pods_util.py]: depends on practices. Some prefer to some don't. Fine I will add spaces around == across the app.

**Comment by breuerfelix** [comment]
changes base to dev,

are these servers already up and running ?

@timgrossmann I don't really know about the pod servers :D so if you approve it we will merge it ! hopefully for 0.6.0 aswell

**Comment by ishandutta2007** [comment]
> are these servers already up and running ?

@breuerfelix Yes

**Comment by ishandutta2007** [comment]
> @timgrossmann I don't really know about the pod servers :D so if you approve it we will merge it ! hopefully for 0.6.0 as well

@breuerfelix  Share me your gmail id, I will add you too ?
I was hoping few people will volunteer to monitor one server/db each. 
once the servers start receiving traffic I will move myself and Tim to editors and only keep help.instapy@gmail.com as the owner.

Can you also do a CR and merge for this https://github.com/InstaPy/instapy-quickstart/pull/62

**Comment by breuerfelix** [comment]
my gmail is f.breuer94@gmail.com , feel free to add me I would love to have a look at that ! really interesting :)

okay if they are up and running I will merge this so it will be released for 0.6.0

**Pull request merged**